docs/logical_data_model.md

```diff
@@ -163,7 +163,8 @@ class ActivityReport {
   ttaType : array<string>
   context : string
   pageState : json
-  managerNotes : string
+  legacyManagerNotes : string
```

Comment:
proposing renaming `managerNotes` to `legacyManagerNotes`

Reply:
I think `legacy` could be confusing, given that label was given to reports imported from Smartsheet. If these fields are going away in the future after a data migration, then this isn't much of a concern.